### PR TITLE
Rewrite doc using RestructuredText

### LINKED ISSUE #210: Making IvoryCKEditorBundle an "official" Symfony bundle
As you can read in [this discussion](https://github.com/symfony/symfony-docs/issues/6248) we'd like to make IvoryCKEditorBundle **the official WYISWYG bundle for Symfony**.

This means that we'll publish this bundle's doc at http://symfony.com/doc/bundles/ ... but that requires to transform the documentation from Markdown to RestructuredText, as used elsewhere in Symfony Docs.

Would you agree to that change? Would you like to do that change yourselves or do you prefer if we create the pull request with the changes?

We'll also add mentions to this bundle wherever is appropriate and makes sense in the Symfony Docs. If you have any idea about where to put these mentions, please tell us.

Thanks for your help!

### PR DESCRIPTION
**Pull request by egeloen**
This PR fixes #210 and is related to symfony/symfony-docs#6248

### CODE REVIEW

**Comment by egeloen** [comment]
@javiereguiluz let me know what do you think. I have build it with sphinx and all is well.

**Comment by javiereguiluz** [comment]
@egeloen thanks for updating the format of your documentation. 95% of your doc is right, but there were some syntax issues. These were mostly related to long lines, so instead of adding comments to this pull request, I've created a new one in #214.

The new pull request ha been created against the `doc` branch, so if everything goes right, you could merge my pull request on top of yours and keep working on your pull request until you merge it in the bundle.

Thanks again for transforming all the documentation!

**Comment by egeloen** [comment]
Thanks as well for your PR! Can you ealoborate about the line length? I have not seen somethink like that in the doc. Is it somethink related to Sphinx or it is only for consistency?

**Comment by javiereguiluz** [comment]
The line length limit and other specific format rules are defined in this doc: http://symfony.com/doc/current/contributing/documentation/standards.html  This limit is not because of Sphinx but because we think this makes the doc more readable in any format and situation (if you read the raw documentation, short lines make it look well).

**Comment by egeloen** [comment]
Sorry, I was searching the `length` keyword in the page and not `char` one... (too lazy to read everything this morning...)

Anyway, thanks for the explanation, let's follow this rule :)

File: Resources/doc/usage/autoinline.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+        ivory_ck_editor:
+            auto_inline: false
+
+    .. code-block:: php
+
+        $builder->add('field', 'ckeditor', array('auto_inline' => false));
```
Comment (WouterJ) [inline Resources/doc/usage/autoinline.rst]: as these 2 examples are showing different things (config vs widget), I recommend to use 2 seperate code blocks/examples. the tabbed code blocks (configuration block) is often used to show the same feature in different formats.

**Comment by egeloen** [comment]
@javiereguiluz I see you remove some `code-block` tags (for example [here](https://github.com/egeloen/IvoryCKEditorBundle/commit/42501aed4bc56db8592303eec286cd366714eb13#diff-44bbac66f4c42b10dbe90f18d46df0e2L41)). Is it intended for?

**Comment by javiereguiluz** [comment]
@egeloen yes. This is because Symfony uses a trick to highlight PHP code without creating a very verbose documentation. I don't agree with this trick (I think it's an ugly and dangerous hack) but it's used everywhere in the Symfony docs, so it's "the official way of doing things".

The trick is to remove `.. code-block:: php` and add two colons `::` in the paragraph previous to the code:

````
BEFORE
If you want to override some parts of the defined config, you can still use the
``config`` option:

.. code-block:: php

    $builder->add('field', 'ckeditor', array(
        'config_name' => 'my_config',


AFTER
If you want to override some parts of the defined config, you can still use the
``config`` option::
  
    $builder->add('field', 'ckeditor', array(
        'config_name' => 'my_config',
```

File: Resources/doc/usage/autoinline.rst (hunk 1 above)
Comment (egeloen) [inline Resources/doc/usage/autoinline.rst]: Done

**Pull request merged**